internal/privatemessaging/privatemessaging_test.go

```diff
@@ -100,6 +96,12 @@ func TestDispatchBatchWithBlobs(t *testing.T) {
 		}
 	}
 
+	mim.On("ResolveInputIdentity", pm.ctx, mock.MatchedBy(func(identity *fftypes.Identity) bool {
+		assert.Equal(t, "org1", identity.Author)
+		identity.Key = "0x12345"
```

Comment:
This fields weird to be altering an arg within a `MatchedBy` method. Should this be a `.Run()` side-effect instead?

Reply:
changed